src/screens/NewTopicScreen.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { postNewTopic}from '../api/DataHandler';
+
+//import { Link } from 'react-router-dom'
+
+interface NewTopicState{
+    post_title: string,
+    post_text: string,
+    post_url: string,
+    username: string,
+    pwdHash: string,
+    hanesst_id: number,
```

Comment:
Does not need hanesst_id, and the pwdHash should just be password.

Reply:
Bedre er at håndtere dem med at sige de kan være: string | undefined

Reply:
Reasoning has been mentioned earlier.

Has been removed